src/time-picker/index.tsx

```diff
@@ -73,6 +92,36 @@ const isMinuteOptionDisabled = (
   (selectedTime.hours === maxTime.hours && i > maxTime.minutes) ||
   (selectedTime.hours === minTime.hours && i < minTime.minutes);
 
+/**
+ * Convert Time to a 12hr TimeDisplay format
+ * @param selectedTime
+ */
+const to12HrTimeDisplay = (selectedTime: Time): TimeDisplay => {
+  let hours;
+  let amPm: any = 'AM';
```

Comment:
You can call this variable `meridiem` instead of amPm and assign it the value of `Meridiem.AM`